openshift/github-mirror-acceptance.yaml

```diff
@@ -4,6 +4,10 @@ kind: Template
 metadata:
   name: github-mirror-acceptance
 objects:
+- apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    name: ${SERVICE_ACCOUNT}
```

Comment:
I think you are also going to need a role that allows this service account to read the `CLIENT_TOKEN_SECRET` secret